go.mod

```diff
@@ -79,13 +79,13 @@ require (
 	go.opentelemetry.io/proto/otlp v0.7.0
 	go.uber.org/zap v1.19.0
 	golang.org/x/crypto v0.0.0-20220315160706-3147a52a75dd
-	golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd
+	golang.org/x/net v0.0.0-20220225172249-27dd8689420f
 	golang.org/x/oauth2 v0.0.0-20211104180415-d3ed0bb246c8
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
-	golang.org/x/sys v0.0.0-20220209214540-3681064d5158
+	golang.org/x/sys v0.0.0-20220319134239-a9b59b0215f8
 	golang.org/x/term v0.0.0-20210927222741-03fcf44c2211
 	golang.org/x/time v0.0.0-20220210224613-90d013bbcef8
-	golang.org/x/tools v0.1.10-0.20220218145154-897bd77cd717
+	golang.org/x/tools v0.1.10
```

Comment:
Is this a downgrade to some older version?

Reply:
it seems there is a v0.1.11 tag https://github.com/golang/tools/commits/v0.1.11

Reply:
> Is this a downgrade to some older version?

I cloned the source, and checked that "v0.1.10" should be newer than "v0.1.10-0.20220218145154-897bd77cd717"
tag "v0.1.10" has the newest commit on top of "v0.1.10-0.20220218145154-897bd77cd717"

On the tag of "v0.1.10", 

> commit 897bd77cd7177f6d679eafa9767ff6ef9211458a (origin)
Author: Robert Findley <rfindley@google.com>
Date:   Fri Feb 18 09:35:21 2022 -0500
>
>    internal/gocommand: remove support for -workfile

    Remove support for passing the -workfile flag now that the go command no
    longer supports it.

    Updates #51215


> it seems there is a v0.1.11 tag https://github.com/golang/tools/commits/v0.1.11

yes,  `v0.1.11` is available, but that should be another topic.

Reply:
Thanks for checking.

/lgtm